Sources/SQLiteNIO/SQLiteDataConvertible.swift

```diff
@@ -43,6 +43,16 @@ extension UInt16: SQLiteDataConvertible { }
 extension UInt32: SQLiteDataConvertible { }
 extension UInt64: SQLiteDataConvertible { }
 
+extension SQLiteData: SQLiteDataConvertible {
+	public init?(sqliteData: SQLiteData) {
+		self = sqliteData
+	}
+
+	public var sqliteData: SQLiteData? {
+		return self
+	}
+}
```

Comment:
Adding this conformance will cause infinite recursion problems in SQLiteKit data marshaling logic. The easiest alternative is usually to special-case values of `SQLiteData` type at the point of use.

Reply:
Fixed